client/components/mma/cancel/cancellationSaves/MembershipCancellationLanding.tsx

```diff
@@ -23,10 +24,9 @@ import { GenericErrorScreen } from '../../../shared/GenericErrorScreen';
 import { JsonResponseHandler } from '../../shared/asyncComponents/DefaultApiResponseHandler';
 import { DefaultLoadingView } from '../../shared/asyncComponents/DefaultLoadingView';
 import { Heading } from '../../shared/Heading';
-import { sectionSpacing } from '../../switch/SwitchStyles';
 import type { CancellationContextInterface } from '../CancellationContainer';
 import { CancellationContext } from '../CancellationContainer';
-import { buttonLayoutCss, headingCss } from './SaveStyles';
+import { buttonLayoutCss } from './SaveStyles';
```

Comment:
would this be a merge conflict with #1171 maybe?

Reply:
It will be a merge conflict but should be fairly straight forward